BaseTools/Source/C/GenFv/GenFvInternalLib.c

```diff
@@ -3837,8 +3837,10 @@ Routine Description:
           //
           // Xip module has the same section alignment and file alignment.
           //
-          Error (NULL, 0, 3000, "Invalid", "PE image Section-Alignment and File-Alignment do not match : %s.", FileName);
-          return EFI_ABORTED;
+          //Error (NULL, 0, 3000, "Invalid", "PE image Section-Alignment and File-Alignment do not match : %s.", FileName);
+          //return EFI_ABORTED;
+          // mu change - avoid forcing DXE core and drivers to be XIP.  
+          continue;

```

Comment:
This is not the path we currently want to go, right? We should take a deeper look here, but can workaround this issue in the meantime.

Reply:
@spbrogan, are we good to abandon this PR?